modules/api/src/main/java/org/apache/ignite/InitParametersBuilder.java

```diff
@@ -93,19 +97,29 @@ public InitParametersBuilder clusterName(String clusterName) {
     }
 
     /**
-     * Sets authentication configuration, that will be applied after initialization. If not set, the authentication will be disabled.
+     * Sets cluster configuration, that will be applied after initialization.
      *
-     * @param authenticationConfig Authentication configuration.
+     * @param clusterConfiguration Cluster configuration.
      * @return {@code this} for chaining.
      */
-    public InitParametersBuilder authenticationConfig(AuthenticationConfig authenticationConfig) {
-        if (authenticationConfig == null) {
-            throw new IllegalArgumentException("Authentication config cannot be null.");
-        }
-        this.authenticationConfig = authenticationConfig;
+    public InitParametersBuilder clusterConfiguration(String clusterConfiguration) {
+        this.clusterConfiguration = clusterConfiguration;
         return this;
     }
 
+    /**
+     * Sets cluster configuration, that will be applied after initialization.
+     *
+     * @param clusterConfiguration Cluster configuration.
+     * @return {@code this} for chaining.
+     */
+    public InitParametersBuilder clusterConfiguration(File clusterConfiguration) throws IOException {
+        try (Stream<String> lines = Files.lines(clusterConfiguration.toPath())) {
+            this.clusterConfiguration = lines.collect(Collectors.joining(System.lineSeparator()));
```

Comment:
Ok, maybe I don't understand something. What's the reason of splitting the file into lines and then concatenating them back?
Wouldn't it be easier to just read the entire file as a string? I'm so confused.